README: Small improvements (#131)
di/id: README.md

<!--- @end-badges@ --->

`id` is a Python tool for generating OIDC identities. It can automatically

> detect and produce OIDC credentials on a number of environments, including
> GitHub Actions, GitLab pipelines and Google Cloud.

Reviewer: Nitpick, nonblocking: maybe link to the "supported environments" section instead of listing a subset here?

```suggestion
detect and produce OIDC credentials in a [number of environments](#supported-environments).
```

Reviewer 2: I'd prefer to keep it as is and highlight the most widely used ones here.

This function requires an `audience` parameter, which is used when generating
the OIDC token. This should be set to the intended audience for the token.

> If no supported environment is found, `detect_credential` returns None. If a supported

Reviewer 2: ```suggestion
If no supported environment is found, `detect_credential` returns `None`. If a supported
```
